org.eclipse.winery.repository.rest/pom.xml

```diff
@@ -505,6 +505,23 @@
 					</executions>
 				</plugin>
 				<plugin>
+					<groupId>org.apache.tomcat.maven</groupId>
+					<artifactId>tomcat7-maven-plugin</artifactId>
```

Comment:
Why not the tomcat8 plugin? Why did you not use IntelliJ to run winery?

The usage of tomcat should be documented in `docs/dev/index.md` (or other fitting place.)